tests/junit.py

```diff
@@ -4,6 +4,7 @@
 import sys
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), 'junit-xml')))
 from junit_xml import TestCase, TestSuite
+from junitparser import JUnitXml
```

Comment:
@jedbrown, the `junit_xml` module does not support merging a preexisting `xml` with current results. This is one module that I found that does. What do you think about adding?

Reply:
It would need to be installed on Noether, LV (easy), and the Azure image. I have verified that this works locally.